docs/2.0 EDID to Receiver Caps mapping.md

```diff
@@ -2,38 +2,41 @@
 
 ## Background
 
-EDID is a metadata format to describe sink devices capabilities to a video source. [AMWA BCP-004-01][BCP-004-01] proposes an approach for a similar purpose in terms of IS-04 Receivers. The goal of this BCP is proposing a mapping scheme of EDID fields to Receiver Caps in order to have a specified way for representing EDID sink devices through associated IS-04 Receivers.
+Extended Display Identification Data (EDID) is a metadata format for display devices to describe their capabilities to a video source (e.g. graphics card or set-top box). The data format is defined by a standard published by the Video Electronics Standards Association (VESA). This document is targeted against [E-EDID A2][E-EDID] files which consist of EDID 1.4 which is called Base EDID and an optional [CTA-861-G][CTA-861] Extension Block.
+
+[AMWA BCP-004-01][BCP-004-01] describes a methodology for a similar purpose in relationship to [AMWA IS-04][IS-04].
 
 ## Scope
 
-This document is targeted against [E-EDID A2][E-EDID] files which consist of EDID 1.4 which is called Base EDID and an optional [CTA-861-G][CTA-861] Extension Block.
+This document provides Best Current Practice guidelines for how to implement a mapping of EDID fields to Receiver Capabilities in order to have a specific way of representing an EDID sink devices through an associated IS-04 Receiver.
 
 ## Use of Normative Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
 
 ## Definitions
 
-_See also the [NMOS Glossary](https://github.com/AMWA-TV/nmos/wiki/Glossary), and definitions within RFCs._
+_See also the [NMOS Glossary](https://specs.amwa.tv/nmos/branches/main/docs/4.0._Glossary.html), and definitions within RFCs._
 
 ### Gateway Device
 
-A device which encapsulates digital video or/and audio (usually, from a physical device) into IP packets or/and does the opposite operation.
-### Sink
+A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) coverted into IP packets. Alternatively, the device may perform the operation reversed converting IP packets to a digital signal.
```

Comment:
Can we stay with this?
```suggestion
A device which encapsulates digital video or/and audio (usually, from a physical connection) into IP packets or/and decapsulates such signals in the reverse way.
```
`Conversion` looks applicable when talking about similar formats for the same purpose which isn't true for IP packets and an arbitrary digital signal. `En-/Decapsulation` shows what exactly happens with the signal if we talk about SMPTE ST 2022/2110 at least.

Another deal is a necessary comment on decapsulating Gateway Devices.
```suggestion
A device which encapsulates digital video or/and audio (usually, from a physical connection) into IP packets or/and decapsulates such signals in the reverse way. Decapsulating Gateway Devices are physically connected to other Sink Devices.
```

Reply:
I like your idea about sticking with encapsulation. I am tweaking it to put the two suggestions together.

```suggestion
A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device may perform the reverse operation for IP packets to a digital signal.
```